initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -100,15 +107,17 @@ if [ -n "${lpath}" -a -h "${lpath}" ] ; then
 fi
 
 if [ -e /etc/debian_version ]; then
-    echo "Removing DEB"
-    apt-get remove -y hpccsystems-platform
+    pckname=$(apt-cache pkgnames hpccsystems-platform)
+    echo "Removing ${pckname}"
+    apt-get remove -y ${pckname} 
```

Comment:
Would there ever be both hpccsystems-platform and hpccsystems-platform-with-spark installed ?
If so would this remove both ?

Reply:
There will always be only one of them. The script will automatically know which one of them is present. Without this fix, it can only remove hpccsystems-platform.